config/services/front/services_prod.yml

```diff
@@ -43,3 +43,11 @@ services:
             - method: queue
               arguments:
                 - ['@prestashop.core.filter.front_end_object.search_result_product']
+
+    prestashop.adapter.module.repository.module_repository:
```

Comment:
Why not putting it in a common file, if you need it in BO and FO?
We have a services/common.yml file

Reply:
It already exists in the Symfony container, besides since 1.7.8 there is no more legacy admin container